packages/@react-aria/separator/docs/useSeparator.mdx

```diff
@@ -78,13 +78,13 @@ function Separator(props) {
 }
 
 <>
-  <div style={{display: 'flex', flexDirection: 'column'}}>
+  <div style={{display: 'flex', direction: 'column'}}>
```

Comment:
Maybe we can keep the flexDirection here? `direction` doesn't accept column or row actually right?
![image](https://user-images.githubusercontent.com/9661127/84960556-cc849980-b0b6-11ea-9b7e-ee93b6552c72.png)